File: SM/Algoritmos, Estrutura de Dados e Linguagens/Exercises/0000000007.pas
PROGRAM ex0000000007;
USES CRT;
VAR peso,pesogordo,pesomagro : real;
BEGIN
    readln(peso);
    pesogordo := peso * 1.15;
    pesomagro := peso - (peso*0.2);
    writeln(pesogordo:3:3);
    writeln(pesomagro:3:3);
END.

File: SM/Algoritmos, Estrutura de Dados e Linguagens/Exercises/0000000007.c
#include <stdio.h>

int main(){

    float peso,pesogordo,pesomagro;
    scanf("%f",&peso);
    pesogordo = peso * 1.15;
    pesomagro = peso - (peso * 0.20);
    printf("%.3f \n %.3f",pesogordo,pesomagro);
}